### ISSUE TITLE
Printing a PDF and TXT fails

### ISSUE THREAD

**Issue by creviera** [freedomofpress/securedrop-export#42]
## Description

I'm seeing a new error when attempting to print a PDF file. The error message in the popup window from the X Printing Panel by Till Kamppeter (December 2004) says:

```
Unable to print file '/tmp/tmpuw_wyzrq/export_data/hello.pdf' - client-error-bad-request.
```

I checked that the file existed on the file system and was able to print using `xpp -P sdw-printer /tmp/tmpuw_wyzrq/export_data/hello.pdf`. I also checked that I could print doc and docx files successfully.

**Comment by creviera** [comment]
whoops, didn't mean to close this

**Comment by creviera** [comment]
Okay, now I am purposely closing this. I'm not sure why I was seeing this error message in the X Printing Panel application earlier, but I am now able to print PDFs and TXTs. I also made sure we can print without file extensions, just to be thorough. I did test with the exact same PDF and TXT file that caused the error a month ago, but since no one else got around to repro'ing, I'll go ahead and close and keep my fingers crossed ;)
